src/components/Payment/Payment.tsx

```diff
@@ -98,8 +111,17 @@ const Payment = ({
                   <small>/{t(`account:periods.${activeSubscription.period}`)}</small>
                 </p>
               </div>
+              <Button
+                className={styles.upgradeSubscription}
+                label={t('user:payment.change_subscription')}
+                onClick={onUpgradeSubscriptionClick}
+                fullWidth={isMobile}
+                color="primary"
+                disabled={!offersAvailable}
+                data-testid="change-subscription-button"
+              />
```

Comment:
Maybe instead of disabling the the button, hide it when there are no switch offers available?